<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>rpms/libxml2.git - libxml2</title>
<meta name='generator' content='cgit v0.12'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/rpms/libxml2.git/atom/libxml2-2.9.4-remove-pyverify_fd.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='https://src.fedoraproject.org/git/rpms/libxml2.git' title='rpms/libxml2.git Git repository'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 5 && tokens[4] == 'rpms' && tokens[5] != '') {
    repo = tokens[5].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='rpms/libxml2.git' href='/cgit/rpms/libxml2.git/'>rpms/libxml2.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f23'>f23</option>
<option value='f24'>f24</option>
<option value='f25'>f25</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>libxml2</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/rpms/libxml2.git/'>summary</a><a href='/cgit/rpms/libxml2.git/refs/'>refs</a><a href='/cgit/rpms/libxml2.git/log/libxml2-2.9.4-remove-pyverify_fd.patch'>log</a><a class='active' href='/cgit/rpms/libxml2.git/tree/libxml2-2.9.4-remove-pyverify_fd.patch'>tree</a><a href='/cgit/rpms/libxml2.git/commit/libxml2-2.9.4-remove-pyverify_fd.patch'>commit</a><a href='/cgit/rpms/libxml2.git/diff/libxml2-2.9.4-remove-pyverify_fd.patch'>diff</a><a href='/cgit/rpms/libxml2.git/stats/libxml2-2.9.4-remove-pyverify_fd.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/rpms/libxml2.git/log/libxml2-2.9.4-remove-pyverify_fd.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/rpms/libxml2.git/tree/'>root</a>/<a href='/cgit/rpms/libxml2.git/tree/libxml2-2.9.4-remove-pyverify_fd.patch'>libxml2-2.9.4-remove-pyverify_fd.patch</a></div><div class='content'>blob: d05d4cb60f3af1503d3c308c3e131e0f483abc92 (<a href='/cgit/rpms/libxml2.git/plain/libxml2-2.9.4-remove-pyverify_fd.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
</pre></td>
<td class='lines'><pre><code>diff -Nur libxml2-2.9.4.orig/python/types.c libxml2-2.9.4/python/types.c
--- libxml2-2.9.4.orig/python/types.c	2016-02-09 03:17:33.000000000 -0700
+++ libxml2-2.9.4/python/types.c	2016-12-21 12:34:06.755650986 -0700
@@ -31,8 +31,6 @@
     const char *mode;
 
     fd = PyObject_AsFileDescriptor(f);
-    if (!_PyVerify_fd(fd))
-        return(NULL);
     /*
      * Get the flags on the fd to understand how it was opened
      */
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='http://git.zx2c4.com/cgit/about/'>cgit v0.12</a> at 2017-01-23 15:15:48 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
